__tests__/__snapshots__/getSnapshotDiffSerializer.test.js.snap

```diff
@@ -5,32 +5,32 @@ Snapshot Diff:
 - First value
 + Second value
 
- Array [
-   "foo",
--  "bar",
-   "baz",
-+  "quoz",
- ]
+  Array [
+    "foo",
+-   "bar",
+    "baz",
++   "quoz",
+  ]
```

Comment:
@SimenB do you have any ideas on why do we have extra space in the snapshots?

Reply:
I'm not sure. It's a diff for jest 21. I'll see if I can isolate it

Reply:
It's this: https://github.com/facebook/jest/commit/7e2c206364bb8d0b3e09b877dd658efc5c27458b#diff-cbe0ce708017e083e342adce28e3720d

/cc @pedrottimark

Reply:
Yes, that PR fixed a difference (pardon the pun :) between options:
* `expand: true` output a one-space “gutter” between -/+/space and content so content aligned with labels (see `can expand diff 1` below)
* `expand: false` did not, but now it does, so the snapshot will not change if you toggle the option (for a diff which has fewer lines than the context)

Reply:
See https://github.com/facebook/jest/pull/3429#issuecomment-326090900

Reply:
You guys are awesome